src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -95,11 +95,7 @@
     public static boolean TEST_BINARY = Boolean.getBoolean(BourneShellScriptTest.class.getName() + ".TEST_BINARY");
     @Parameters(name = "{index}: {0}")
     public static Object[] data() {
-        if (Objects.equals(System.getenv().get("SKIP_DURABLE_TASK_BINARY_GENERATION"), "true")) {
-            return new TestPlatform[]{TestPlatform.NATIVE, TestPlatform.ALPINE, TestPlatform.CENTOS, TestPlatform.UBUNTU, TestPlatform.NO_INIT, TestPlatform.SLIM};
-        } else {
-            return TestPlatform.values();
-        }
+        return TestPlatform.values();
```

Comment:
Trying to follow the decisions in #106 & #121. If Docker is not working but we are in CI (typically because we are running in Windows), native image generation will be skipped…as will this whole test: https://github.com/jenkinsci/durable-task-plugin/blob/6b6a88ee2b2a8bf5019d67f05e28a4611dd03efa/src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java#L112

Reply:
On `maven-windows`:

```
 [INFO] Running org.jenkinsci.plugins.durabletask.BourneShellScriptTest
 [WARNING] Tests run: 1, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.011 s - in org.jenkinsci.plugins.durabletask.BourneShellScriptTest
```

As expected?

Reply:
And mostly passing now on Linux, with just a couple skips, though `BourneShellScriptTest` is _really_ slow.